saleor/plugins/base_plugin.py

```diff
@@ -237,15 +237,6 @@ def show_taxes_on_storefront(self, previous_value: bool) -> bool:
         """
         return NotImplemented
 
-    def apply_taxes_to_shipping_price_range(
```

Comment:
Should we add this to `breaking changes`?

Reply:
When 3.0 is released, probably yes. Now there is no "breaking changes" section in the changelog for 3.0. I can also create it.

Reply:
We should keep one updated, so we don't have to reconstruct it from memory at launch.

Reply:
Added "Braking" section in the changelog.